testsuite/features/support/commonlib.rb

```diff
@@ -41,7 +41,7 @@ def compute_image_filename
   case ENV['PXEBOOT_IMAGE']
   when 'sles15sp3', 'sles15sp3o'
     # 'Kiwi/POS_Image-JeOS7_42' for 4.2 branch
-    'Kiwi/POS_Image-JeOS7_head'
+    $product == 'Uyuni' ? 'Kiwi/POS_Image-JeOS7_uyuni' : 'Kiwi/POS_Image-JeOS7_head'
```

Comment:
You do this change only for JeOS7, but not JeOS6 which is also copied in this PR.
If you do not need JeOS6 (SLE12 based) to test, I suggest to drop JeOS6 copies in this PR.